src/components/manage/Toast/toastComponent/toast.js

```diff
@@ -0,0 +1,23 @@
+import { toast as toastify } from 'react-toastify';
+import config from '@plone/volto/registry';
```

Comment:
This whole file feels like it should belong helpers folder.

Reply:
sounds good